src/test/java/org/folio/edge/rtac/utils/RtacOkapiClientTest.java

```diff
@@ -117,7 +118,7 @@ public void testRtacTimeout(TestContext context) throws Exception {
       future.get();
       fail("Expected a TimeoutException to be thrown");
     } catch (Exception e) {
-      assertEquals(TimeoutException.class, e.getCause().getClass());
+      assertTrue(e.getMessage().contains("TimeoutException"));
     }
```

Comment:
Vertx four does not seem to trigger java timeoutExceptions as responses anymore-the actual timeout error returned is now a vertx timeout error with a very lengthy string message.